projects/vdk-core/src/vdk/internal/builtin_plugins/config/vdk_config.py

```diff
@@ -113,6 +114,15 @@ def vdk_configure(self, config_builder: ConfigurationBuilder) -> None:
             "Data Job execution can run a Data Job one or more times."
             "Each distinct run would a single attempt.",
         )
+        config_builder.add(
+            WRITE_DIRECTORY,
+            "/var/tmp",
+            True,
+            "Write data job directory, to be used if job needs to write files"
+            " to local storage during cloud execution (since writing to any "
```

Comment:
Let's make it explicit that any file saved there would be automatically cleaned after the job execution finishes.